node/src/service.rs

```diff
@@ -87,7 +65,8 @@ pub fn new_partial(
 	>,
 	sc_service::Error,
 > {
-	let inherent_data_providers = build_inherent_data_providers(author)?;
+	let inherent_data_providers =
+		crate::inherents::build_inherent_data_providers(author, false, false)?;
```

Comment:
are there ever situations when this function passes in `(author, true, false)` or `(author, false, true)`? if not, the bool inputs could just be one input `mock: bool`

Reply:
Done in f9abac1